set.mm

```diff
@@ -615019,6 +615019,92 @@ fixed reference functional determined by this vector (corresponding to
 #*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
 $)
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+Utility theorems
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    ioin9i8.1 $e |- ( ph -> ( ps \/ ch ) ) $.
+    ioin9i8.2 $e |- ( ch -> -. th ) $.
+    ioin9i8.3 $e |- ( ps -> th ) $.
+    $( Miscellaneous inference creating a biconditional from an implied
+       converse implication.  (Contributed by Steven Nguyen, 17-Jul-2022.) $)
+    ioin9i8 $p |- ( ph -> ( ps <-> th ) ) $=
+      ( wn ord syl6 con4d impbid2 ) ABDGABDABHCDHABCEIFJKL $.
+  $}
+
+  ${
+    jaodd.1 $e |- ( ph -> ( ps -> ( ch -> th ) ) ) $.
+    jaodd.2 $e |- ( ph -> ( ps -> ( ta -> th ) ) ) $.
+    $( Double deduction form of ~ jaoi .  (Contributed by Steven Nguyen,
+       17-Jul-2022.) $)
+    jaodd $p |- ( ph -> ( ps -> ( ( ch \/ ta ) -> th ) ) ) $=
+      ( wi wo jao syl6c ) ABCDHEDHCEIDHFGCDEJK $.
+  $}
+
+  ${
+    $d ph x y z $.  $d ch x $.  $d th y $.  $d ta z $.  $d D x y z $.
+    $d A x y z $.  $d B y z $.  $d C z $.
+    3rspcedvd.a $e |- ( ph -> A e. D ) $.
+    3rspcedvd.b $e |- ( ph -> B e. D ) $.
+    3rspcedvd.c $e |- ( ph -> C e. D ) $.
+    3rspcedvd.1 $e |- ( ( ph /\ x = A ) -> ( ps <-> ch ) ) $.
+    3rspcedvd.2 $e |- ( ( ph /\ y = B ) -> ( ch <-> th ) ) $.
+    3rspcedvd.3 $e |- ( ( ph /\ z = C ) -> ( th <-> ta ) ) $.
+    3rspcedvd.4 $e |- ( ph -> ta ) $.
+    $( Triple application of ~ rspcedvd .  (Contributed by Steven Nguyen,
+       27-Feb-2023.) $)
+    3rspcedvd $p |- ( ph -> E. x e. D E. y e. D E. z e. D ps ) $=
+      ( wrex cv wceq wa 2rexbidv rexbidv rspcedvd ) ABHLTGLTCHLTZGLTFILMAFUAIUB
+      UCBCGHLLPUDAUGDHLTGJLNAGUAJUBUCCDHLQUEADEHKLORSUFUFUF $.
+  $}
+
+  $( The proper subset of a set is also a set.  (Contributed by Steven Nguyen,
+     17-Jul-2022.) $)
+  pssexg $p |- ( ( A C. B /\ B e. C ) -> A e. _V ) $=
+    ( wpss wss wcel cvv pssss ssexg sylan ) ABDABEBCFAGFABHABCIJ $.
+
+  $( A proper superset is nonempty.  (Contributed by Steven Nguyen,
```

Comment:
```suggestion
  $( A proper superclass is nonempty.  (Contributed by Steven Nguyen,
```

Reply:
https://us.metamath.org/mpeuni/df-ss.html says subset is used instead of subclass, so it kinda applies to supersets too.

ctrl+f finds 63 "superset" and 6 "superclass" (and 1378 "subset" and 374 "subclass")

Reply:
As you prefer.